### ISSUE TITLE
🐛[BUG] lost all context

### ISSUE THREAD

**Issue by giovannicuce** [ant-design/ant-design-charts#2745]
Hi,

I have a problem with library "@ant-design/graphs": "2.0.0-beta.4"

``<OrganizationChart
          key={keyChart}
          loading={isLoading}
          data={data}
          zoom={1}
          x={position.x}
          y={position.y}
          height={heightChart}
          padding={[40, 0, 0, 120]}
          edge={{
            style: {
              radius: 16,
              lineWidth: 2,
              endArrow: true,
            },
          }}
          layout={{
            type: "antv-dagre",
            nodesep: -10,
            ranksep: 80,
          }}
          transforms={(prev) => [
            ...prev.filter(
              (transform) => transform.type !== "collapse-expand-react-node",
            ),
            {
              ...prev.find(
                (transform) => transform.type === "collapse-expand-react-node",
              ),
              enable: true,
              iconOffsetX: 12,
            },
          ]}
          direction={"horizontal"}
          node={{
            style: {
              component: (d: Values) => {
                return (
                  <OrganizationItem
                    object={d.data}
                    setInitialValue={setInitialValue}
                    initialValue={initialValue}
                    setKeyChart={setKeyChart}
                    style={d.style}
                    setPosition={setPosition}
                    widthChart={widthChart}
                    heightChart={heightChart}
                    calculateSizeItem={calculateSizeItem}
                    setData={setData}
                  />
                );
              },
              //@ts-ignore
              size: (element: Values) => {
                return initialValue?.id == element.id
                  ? [240, 350]
                  : calculateSizeItem(element.data);
              },
            },
          }}
        />``

In this code the component OrganizationItem lost all context of the app. Is there a way to avoid losing them?

**Comment by yvonneyx** [comment]
If you want to get the context outside the graph component, you can use `ref`. 

```jsx
export default () => {
    const graphRef = useRef();

   return <OrganizationChart ref={(ref) => { graphRef.current = ref; }}  />
}
```

If you want to get the context in a configuration item in options, you can use this method~

```jsx
export default () => {
    const options = {
         node: { style: { size: function() { console.log(this)} } }
    }

   return <OrganizationChart {...options} />
}
```

**Comment by giovannicuce** [comment]
Sorry, I explained myself poorly. We created a custom context (OrganizationContext) for our application, along with the corresponding hook. We need to access OrganizationContext through the hook useOrganizationContext within the custom render of the node, but the context seems empty. How can we resolve this issue?

We have encountered similar behavior with the AntProComponents contexts: the form and various fields are unable to take the localization (EN) from the context, always displaying the language as ZH.
